rs/nervous_system/tla/src/lib.rs

```diff
@@ -0,0 +1,25 @@
+use ic_nns_constants::GOVERNANCE_CANISTER_ID;
+use icp_ledger::{AccountIdentifier, Subaccount};
+
+pub use tla_instrumentation::{Destination, TlaValue, ToTla};
+
+pub mod macros;
+
+#[cfg(feature = "tla")]
```

Comment:
why does the tla crate have a tla feature?  It seems like the TLA feature should always be assumed to be on if the TLA crate is being imported?

Reply:
It's just an extra safeguard that we don't turn the logging on in prod by accident. The `store` is essentially where all the logging data goes; this part should most definitely never be used in prod. 

The `macros` module is not gated behind the feature flag, since the macros expand to no-ops unless the feature is turned on. The alternative would be to wrap every use site of the macros behind the `#[cfg(tla)]` directive. I went with the former as I was trying to keep the affected prod LOC to minimum but happy to change if you think it's better.